guides/v2.0/config-guide/cli/config-cli-subcommands-mode.md

```diff
@@ -126,12 +126,17 @@ When you change from production to developer mode, you should clear generated cl
 
 2.  Set the mode:
 
-		magento deploy:mode:set developer
+		bin/magento deploy:mode:set developer
 
 	The following message displays:
 
 		Switched to developer mode.
 
+### To run Magento commands from any directory
+<a href="{{ page.baseurl }}config-guide/cli/config-cli.html#config-install-cli-first">Run Magento system commands from anywhere</a>
```

Comment:
Suggest rewrite and add period at end of sentence:

> Run Magento **CLI commands** from anywhere.

Also, please use markdown syntax for links.